assets/sass/style.scss

```diff
@@ -1,7 +1,99 @@
 @import 'reset';
+html {
+  height: 100%;
+}
 body {
   margin: 0;
+  height: 100%;
+  width: 100%;
```

Comment:
Please recheck the font family and sizes used in the page. Seems very much different from the provided design.